scripts/exregional_get_obs_nohrsc.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+
+#
+#-----------------------------------------------------------------------
+#
+# Source the variable definitions file and the bash utility functions.
+#
+#-----------------------------------------------------------------------
+#
+. $USHdir/source_util_funcs.sh
+source_config_for_task " " ${GLOBAL_VAR_DEFNS_FP}
+#
+#-----------------------------------------------------------------------
+#
+# Save current shell options (in a global array).  Then set new options
+# for this script/function.
+#
+#-----------------------------------------------------------------------
+#
+{ save_shell_opts; . $USHdir/preamble.sh; } > /dev/null 2>&1
+#
+#-----------------------------------------------------------------------
+#
+# This script retrieves and organizes the NOHRSC data into a more intuitive structure:
+# A valid YYYYMMDD directory is created, and all files for the valid day are placed within the directory.
+# NOTE: Accumulation is currently hardcoded to 06h and 24h which are available every 6 and 12 hours, respectively.
+#
+#-----------------------------------------------------------------------
+#
+
+set -x
```

Comment:
Probably should remove this `set -x` since it adds a lot of clutter to the log file.  User can add it when debugging (or you can put this in an if-statement on the `DEBUG` or similar flag).